--- libs/python/src/object/class.cpp	2018-05-03 09:52:01.070585239 +0200
+++ libs/python/src/object/class.cpp.p	2018-05-03 10:04:38.530665696 +0200
@@ -618,7 +618,7 @@
   {
       object property(
           (python::detail::new_reference)
-          PyObject_CallFunction((PyObject*)&PyProperty_Type, const_cast<char*>("Osss"), fget.ptr(), 0, 0, docstr));
+          PyObject_CallFunction((PyObject*)&PyProperty_Type, const_cast<char*>("Osss"), fget.ptr(), (char*)NULL, (char*)NULL, docstr));
       
       this->setattr(name, property);
   }
@@ -628,7 +628,7 @@
   {
       object property(
           (python::detail::new_reference)
-          PyObject_CallFunction((PyObject*)&PyProperty_Type, const_cast<char*>("OOss"), fget.ptr(), fset.ptr(), 0, docstr));
+          PyObject_CallFunction((PyObject*)&PyProperty_Type, const_cast<char*>("OOss"), fget.ptr(), fset.ptr(), (char*)NULL, docstr));
       
       this->setattr(name, property);
   }
